thirdeye/thirdeye-frontend/app/pods/rootcause/template.hbs

```diff
@@ -1,3 +1,23 @@
+{{#if hasErrors}}
+  <div class="alert alert-danger alert-dismissable fade in">
+    <a class="close" data-dismiss="alert" {{action "clearErrors"}}>&times;</a>
+    <ul>
+      {{#if entitiesService.errors.size}}
+        <li>Error for task(s):{{#each entitiesService.errors as |e|}} <strong>{{e.framework}}</strong>{{/each}}</li>
+      {{/if}}
+      {{#if timeseriesService.errors.size}}
+        <li>Could not load timeseries:{{#each timeseriesService.errors as |e|}} <strong>{{e.urn}}</strong>{{/each}}</li>
```

Comment:
I can see a pattern here, perhaps an opportunity to componentized later on (obviously we'll need a design first :) )